package.json

```diff
@@ -2,6 +2,8 @@
   "name": "design-system",
   "private": true,
   "scripts": {
+    "postinstall": "lerna bootstrap",
+    "build:cdn": "lerna run --scope @livechat/design-system build:cdn",
```

Comment:
"postinstall" and  "build:cdn" - second needed change

Reply:
what's wrong here?

Reply:
nothing, I meant that this change is needed xD